bundle/README.md

```diff
@@ -256,6 +256,12 @@ To install Prometheus you will need to follow this steps:
         ```sh
         $ oc get route kiali -o jsonpath='{.spec.host}' -n istio-system
         ```
+## Validating the configuration
+
+The Operator validates the contents of the `spec.values` parameter. When parsing parameter contents, if the Operator encounters an unknown value, it generates an error. To force the Operator to pass an unknown value, use the `spec.rawValues` field.
+
+You can use the `oc explain` command to display the valid values for custom fields. For example, `oc explain istio.spec.values`.
```

Comment:
you can replace `custom fields` with something like `the fields in the Istio custom resource`

Reply:
@jwendell would it make sense to say custom resource definition?

Something like "...to display the valid values for the fields in the Istio custom resource definition."

Reply:
yeah, or just `CRD`, or just `Istio resource`

Reply:
Done